ui/page/start_page.go

```diff
@@ -61,7 +62,12 @@ func (sp *startPage) OnNavigatedTo() {
 		if sp.WL.MultiWallet.IsStartupSecuritySet() {
 			sp.unlock()
 		} else {
-			go sp.openWallets("")
+			go func() {
+				if err := sp.openWallets(""); err != nil {
+					sp.Toast.NotifyError(fmt.Sprintf("Error opening wallet: %v", err))
+					// TODO: Exit app?
+				}
+			}()
```

Comment:
Silently dealing with any error that occurs here causes the app to stay at the "Opening Wallets" page with no GUI indication of an error. This fixes that, e.g:

![Screenshot 2022-01-22 at 05 56 13](https://user-images.githubusercontent.com/18400051/150625505-b12c0d78-795a-4665-b1b5-c962ed28db28.png)

Reply:
Remove the commit with that change.